NuRadioReco/modules/ARA/triggerSimulator.py

```diff
@@ -200,49 +247,46 @@ def run(self, evt, station, det,
 
         # No coincidence requirement yet
         trigger = {}
+        trigger_times = []
         for channel in station.iter_channels():
             channel_id = channel.get_id()
             if channel_id not in triggered_channels:
                 continue
             trigger[channel_id] = self.has_triggered(channel)
+            if trigger[channel_id]:
+                times = channel.get_times()
+                trace_after_diode = self.tunnel_diode(channel)
+                arg_trigger = np.argwhere( trace_after_diode == np.min(trace_after_diode) )[0,0]
+                trigger_times.append(times[arg_trigger])
 
         has_triggered = False
-        trigger_time_sample = None
-        # loop over the trace with a sliding window of "coinc_window"
-        coinc_window_samples = np.int(np.round(coinc_window * channel.get_sampling_rate()))
-        trace_length = len(station.get_channel(0).get_trace())
-        sampling_rate = station.get_channel(0).get_sampling_rate()
-
-        for i in range(0, trace_length - coinc_window_samples):
-            istop = i + coinc_window_samples
-            coinc = 0
-            trigger_times = []
-            for iCh, tr in trigger.items():  # loops through triggers of each channel
-                tr = np.array(tr)
-                mask_trigger_in_coind_window = (tr >= i) & (tr < istop)
-                if(np.sum(mask_trigger_in_coind_window)):
-                    coinc += 1
-                    trigger_times.append(tr[mask_trigger_in_coind_window][0])  # save time/sample of first trigger in
-
-            if coinc >= number_concidences:
+        trigger_time = None
+
+        trace_times = station.get_channel(0).get_times()
```

Comment:
There may not be a channel with ID 0.
As addressed in #160 , I don't think we should assume that all channels have the same trace_start_time, since there may be differences between the deep and shallow channels or if we have a double trigger for DnR

Reply:
That was in the module before. Let me change it.

Reply:
Keep in mind that the assumption that they have the same trace_start_time is nowhere. I get, for every channel, the correct trigger times array given by the get_times() method. I agree that the way of sliding the window is not the best, as it should go from the minimum time of all the channels, to the maximum time.

Reply:
Done. Let me know if it's better this way.